Added Accrualify section (#574)
mattermost/mattermost-handbook: operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md

## How to navigate Accrualify

> When you log into Accrualify, you'll see your dashboard. On the upper right hand corner, you can see your avatar.

Reviewer: ```suggestion
When you log in to Accrualify, you're taken directly to your dashboard. In the upper right hand corner of the screen, you can see your avatar.
```

![](../../../../.gitbook/assets/How-to-navigate-Accrualify-1.PNG)

> From here you can update your profile and access the Accrualify user manual.

Reviewer: ```suggestion
Select your avatar to update your user profile and to access the Accrualify user manual.
```

![](../../../../.gitbook/assets/How-to-navigate-Accrualify-2.PNG)

> On the left hand side, you'll see the different modules you have access to and can toggle between.

Reviewer: ```suggestion
In the left hand navigation pane you'll see the different modules you can access. Some of the common modules you will use are described below.
```

![](../../../../.gitbook/assets/How-to-navigate-Accrualify-3.PNG)

> In the **Vendor** module, you'll see the list of vendors that Mattermost purchases from. To search for vendors, you can type the name of the vendor in the text box. To add columns for other information, click on the green plus sign to bring down the columns menu. From here you are able to add additional columns on the grid, such as **ID**, **Account manager**, and **Terms**.

Reviewer: ```suggestion
Select the **Vendor** module to see the list of vendors that Mattermost purchases from. 
- To search for a vendor, enter the name of the vendor in the **Vendor** text box. 
- To display more vendor details, select the green plus sign to the far right of the screen to see available column options. From here, select the columns to display on the grid, such as **Id**, **Acct Mgr**, and **Terms**.
```

![](../../../../.gitbook/assets/How-to-navigate-Accrualify-4.PNG)

![](../../../../.gitbook/assets/How-to-navigate-Accrualify-5.PNG)

> In the **Purchases** module, you'll see the list of purchase orders that you have submitted.

Reviewer: ```suggestion
Select the **Purchases** module to see purchase orders that you've submitted.
```

![](../../../../.gitbook/assets/How-to-navigate-Accrualify-6.PNG)

> Within the **Purchases** module, you'll see the list of purchase orders that are on your queue for review and approval.

Reviewer: ```suggestion
Within the **Purchases** module, you'll also see purchase orders ready for your review and approval.
```

![](../../../../.gitbook/assets/How-to-navigate-Accrualify-7.PNG)

> In the **Invoices** module, you will see all the invoices that are on your queue for approval.

Reviewer: ```suggestion
Select the **Invoices** module to see invoices that are ready for your approval.
```

![](../../../../.gitbook/assets/How-to-navigate-Accrualify-8.PNG)

> In the **Credit memos** module, you will also be able to see any credit memos that would need to be applied to an invoice. This would probably not happen too often.

Reviewer: ```suggestion
Select the **Credit Memos** module to see any credit memos that need to be applied to an invoice. This is something that you'll do infrequently.
```

![](../../../../.gitbook/assets/How-to-navigate-Accrualify-9.PNG)

In the **Payments** module, you'll see the payment status of your invoices.

![](../../../../.gitbook/assets/How-to-navigate-Accrualify-10.PNG)

> At this point in time, you will not need to use the Accruals module and the Corporate Cards module. For Corporate Card requests, please refer to the [“Virtual Card”](https://handbook.mattermost.com/operations/finance/spending-company-money/procurement/request-a-divvy-virtual-credit-card) section of the Handbook.

Reviewer: ```suggestion
For now, you won't need to access the **Accruals** or **Corporate Cards** modules.

For Corporate Card requests, please refer to the [Virtual Credit Card](https://handbook.mattermost.com/operations/finance/spending-company-money/procurement/request-a-divvy-virtual-credit-card) section of the Handbook.
```
